src/core/project/qgsproject.h

```diff
@@ -2047,7 +2047,7 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      *
      * \since QGIS 3.26
      */
-    bool rollBack( bool stopEditing = true, QgsVectorLayer *vectorLayer = nullptr );
+    bool rollBack( QStringList &rollbackErrors, bool stopEditing = true, QgsVectorLayer *vectorLayer = nullptr );
```

Comment:
Should this be `SIP_OUT` (that will add it to the return value in python)?

Reply:
Yes it would make sense, i was not aware of this